create-turbo/src/shouldUsePnpm.ts

```diff
@@ -15,5 +15,5 @@ export function shouldUsePnpm(): boolean {
 
 export function getNpxCommandOfPnpm() {
   const currentVersion = execSync("pnpm --version").toString();
-  return Number(currentVersion.charAt(0)) >= 6 ? "pnpm dlx" : "pnpx";
+  return Number(currentVersion.split(".")[0]) >= 6 ? "pnpm dlx" : "pnpx";
```

Comment:
wrap this in a try catch and throw an Error if the command fails (i.e. if it used but pnpm is not installed)